src/lib/net/FingerprintDatabase.cpp

```diff
@@ -7,57 +7,71 @@
 
 #include "FingerprintDatabase.h"
 
-#include "base/String.h"
-#include "io/Filesystem.h"
+#include <QFile>
 
+#include <QTextStream>
```

Comment:
Was the split intentional?

```suggestion
#include <QFile>
#include <QTextStream>
```